docs/toolsets.md

```diff
@@ -598,6 +598,8 @@ To register a dynamic toolset, you can pass a function that takes [`RunContext`]
 
 By default, the function will be called again ahead of each agent run step. If you are using the decorator, you can optionally provide a `per_run_step=False` argument to indicate that the toolset only needs to be built once for the entire run.
 
+When using [Temporal durable execution](./durable_execution/temporal.md), the decorator also accepts an `id` parameter to uniquely identify the toolset. If not provided, the function name is used as the ID.
```

Comment:
This makes it seem like it only accepts the parameter when Temporal is used, while of course it always accepts the parameter :) 

If we don't use the function name by default as I suggested above, we may not need this line here at all, as this won't be relevant unless they use Temporal, and if they do they'll get an error or read it in the docs